src/styles/rating-stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  display: flex;
+
+  &__star {
+    display: flex;
+    width: 16px;
+    height: 16px;
+    background: no-repeat center url(/src/images/star.svg);
+    margin-right: 4px;
+  }
+}
+
+.stars--1 .stars__star:nth-child(-n + 1),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background-image: url(/src/images/star-active.svg);
```

Comment:
Similar to the previous comment, consider using a variable for the URL path to the active star image for better maintainability.